### PR TITLE
Organize expression and render test files

### PR DESCRIPTION
**Pull request by birkskyum**
Clean up of the integration tests.

Render and Expression test suites now have individual folders inside of the integration directory, rather than having files scattered around the test dir.

### CODE REVIEW

**Comment by birkskyum** [comment]
For some reason the GitHub workflows are not triggered - will investigate

**Comment by wipfli** [comment]
Same in the docs repo. Maybe another GitHub thing...

**Comment by birkskyum** [comment]
Seems likely https://www.githubstatus.com - degraded webhooks and actions

File: test/integration/render/suite_implementation.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import {PNG} from 'pngjs';
 import request from 'request';
-import maplibregl from '../rollup/build/tsc/src/index';
-import browser from '../rollup/build/tsc/src/util/browser';
-import * as rtlTextPluginModule from '../rollup/build/tsc/src/source/rtl_text_plugin';
+import maplibregl from '../../../rollup/build/tsc/src/index';
```
Comment (HarelM) [inline test/integration/render/suite_implementation.ts]: Can we use the regular files and not the rollup output since this is now in typescript?

**Comment by HarelM** [comment]
It's extremely hard to review this since there's too many file changes.
Since I haven't seen anything special and this PR is mainly folder move I guess this can be approved.
We need to see which files are still using the rollup output folder and fix this as a final step related to typescript migration.

**Review by HarelM — APPROVED** [review]

File: test/integration/render/suite_implementation.ts (hunk 1 above)
Comment (birkskyum) [reply test/integration/render/suite_implementation.ts]: Definitely - I will check for this throughout

**Comment by birkskyum** [comment]
I agree, I didn't like the pr to be this big, but I don't know how else to move these folders around without blowing it up.

**Comment by HarelM** [comment]
Can you make a short commit to see if will trigger the CI?
In any case, as I said, this LGTM.

**Comment by birkskyum** [comment]
Yes, we definitely need all the workflows running in order merge this safely

**Comment by wipfli** [comment]
> It's extremely hard to review this since there's too many file changes.

If a pull request has a few manual changes and many automatic changes done with a tool, then it is nice when the manual changes are in one commit and the automatic ones are in another commit. Like this I can review the manual changes and then make the automatic changes myself and check that the diff to your branch is empty.

**Comment by wipfli** [comment]
That said, I trust you @birkskyum that your pull request is fine :)

**Pull request merged**